ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobs.tsx

```diff
@@ -264,12 +265,19 @@ function ReportJobs({ reportId }: RunHistoryProps) {
                                             : '-'}
                                     </Td>
                                     <Td dataLabel="Status">
-                                        <ReportJobStatus reportSnapshot={reportSnapshot} />
+                                        <ReportJobStatus
+                                            reportSnapshot={reportSnapshot}
+                                            areDownloadActionsDisabled={!areDownloadActionsDisabled}
```

Comment:
Why the negation of `areDownloadActionsDisabled` here and below? If it isn't an error, it is somewhat confusing when reading the code since it seems to communicate the opposite of what is expected.

Reply:
Oops. I was testing the opposite condition to double-check how it looked like. Forgot to set it back